packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -182,6 +182,10 @@ function _SignUpStart(): JSX.Element {
       [] as Array<FormControlState>,
     );
 
+    if (unsafeMetadata) {
+      fieldsToSubmit.push({ id: 'unsafeMetadata', value: unsafeMetadata } as any);
```

Comment:
I believe the changes in this file should also be included in ` <SignUpContinue />`